deployment/addons/nginx-app-ingress/hybrid-java-ingress.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
```

Comment:
is this ingress for the java app? Can we move it under applications/java/deployment along with the rest of k8s manifest for this app?

Reply:
Yes, moved!